QModPluginEmulator/QModPluginGenerator.cs

```diff
@@ -88,30 +89,26 @@ public static void Finish()
             "PerformGarbage", "Fallback handler could not load"
         };
 
-        public static List<string> DirtyMidStrings = new List<string>()
+        public static List<string> DirtyPatterns = new List<string>()
```

Comment:
Could this be `readonly`?

Reply:
Another fair point. I'll change the other List in this codebase to `readonly` while I'm at it.

Reply:
e2e2a8f